ktor-client/ktor-client-core/common/src/io/ktor/client/features/cache/HttpCacheEntry.kt

```diff
@@ -82,8 +82,12 @@ internal fun HttpResponse.cacheExpires(): GMTDate {
         return call.response.requestTime + maxAge * 1000L
     }
 
-    headers[HttpHeaders.Expires]?.fromHttpToGmtDate()?.let { return it }
-    return GMTDate()
+    try {
+        val expires = headers[HttpHeaders.Expires]!!.fromHttpToGmtDate() 
+        return expires
+    } catch(e: Throwable) {
```

Comment:
We definitely shouldn't simply ignore the error. I would skip missing or "blank" content but crash at an invalid content.